### ISSUE TITLE
Comments with newlines are not formatted nicely in createPropertyFormatter

### ISSUE THREAD

**Issue by jorenbroekema** [amzn/style-dictionary#952]
This was raised in our repo https://github.com/tokens-studio/sd-transforms/issues/62 but I believe it to be best fixed in style-dictionary `createPropertyFormatter`.

## Current

```css
:root {
  --foo-bar: #FFFFFF; // Some comment with a 
<-- new line here that creates invalid CSS
}
```

^ this part I would call a bug, because it actually results in broken CSS, so in that sense this issue is not just a feature request but moreso a bug report.

or if you set it to commentStyle `"long"`:

```css
:root {
  --foo-bar: #FFFFFF; /* Some comment with a 
<-- new line here that creates invalid CSS */
}
```

## Expected

Newlines to be filtered out in short:

```css
:root {
  --foo-bar: #FFFFFF; // Some comment with a <-- new line here that creates invalid CSS
}
```

or with commentStyle `"long"`:

```css
:root {
  /** 
   * Some comment with a 
   * <-- new line here that creates invalid CSS
   */
  --foo-bar: #FFFFFF;
}
```

I'm happy to create a PR for this, but only if you guys have the time to review & release it at some point :)

**Comment by chazzmoney** [comment]
@jorenbroekema We will have time to review and release - we are checking once a week or so right now. If you have a PR submitted, we will take a look.
